### PR TITLE
make an InvoiceSummary class

### PR DESCRIPTION
**Pull request by stephenwaite**
surprisingly few uses and many unnecessary requires

### CODE REVIEW

File: src/Billing/InvoiceSummary.php [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+
+require_once(dirname(__FILE__) . "/../../custom/code_types.inc.php");
+
+use OpenEMR\Billing\SLEOB;
+
+// for Integrated A/R.
+//
+class InvoiceSummary
+{
+    public function ar_get_invoice_summary($patient_id, $encounter_id, $with_detail = false)
```
Comment (bradymiller) [inline src/Billing/InvoiceSummary.php]: this is a static function:
```
public static function ar_get_invoice_summary($patient_id, $encounter_id, $with_detail = false)
```

File: src/Billing/InvoiceSummary.php [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+        }
+
+        return $codes;
+    }
+
+// This determines the party from whom payment is currently expected.
+// Returns: -1=Nobody, 0=Patient, 1=Ins1, 2=Ins2, 3=Ins3.
+// for Integrated A/R.
+//
+    public function ar_responsible_party($patient_id, $encounter_id)
```
Comment (bradymiller) [inline src/Billing/InvoiceSummary.php]: this is a static function:
```
public static function ar_responsible_party($patient_id, $encounter_id)
```

**Comment by bradymiller** [comment]
class cheer :)
crowd

**Comment by stephenwaite** [comment]
lol, i like the guy in the lower right corner who needs help from the usher

**Comment by stephenwaite** [comment]
testing fine

**Comment by bradymiller** [comment]
hi @stephenwaite , Looks good and rec bringing this PR in as a "Squash and merge", which will remove the merge commit and make it all one commit. Thanks!

**Pull request merged**